cmd/kueueviz/frontend/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "kueueviz-frontend",
-  "version": "1.0.0",
+  "version": "0.12.3",
```

Comment:
Why not `v0.12.3`?

Reply:
If I understood correctly, it should follow semantic versioning.

https://docs.npmjs.com/cli/v11/configuring-npm/package-json#version
https://docs.npmjs.com/about-semantic-versioning
https://semver.org/

Reply:
Oh, I see. So, we should use the semver w/o `v`.
LGTM, thank you.